src/new-relic/azext_new_relic/tests/latest/test_new_relic.py

```diff
@@ -0,0 +1,89 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest,ResourceGroupPreparer)
+
+
+class NewRelicScenario(ScenarioTest):
+
+    @AllowLargeResponse(size_kb=10240)
+    @ResourceGroupPreparer(name_prefix='cli_test_new_relic_monitor')
+    def test_new_relic_monitor(self, resource_group):
+        self.kwargs.update({
+            'new_relic_monitor_name': 'test-new-relic-monitor',
+            'user_email': 'v-taoxuzeng@microsoft.com',
```

Comment:
You should consider use a fake email if the fake email can work.

Reply:
Fake email cannot create success.